graylog2-server/src/main/java/org/graylog2/indexer/messages/Messages.java

```diff
@@ -25,6 +25,15 @@
 import com.github.rholder.retry.WaitStrategies;
 import com.google.common.base.Throwables;
 import com.google.common.collect.ImmutableMap;
+import io.searchbox.client.JestClient;
+import io.searchbox.client.JestResult;
+import io.searchbox.core.Bulk;
+import io.searchbox.core.BulkResult;
+import io.searchbox.core.DocumentResult;
+import io.searchbox.core.Get;
+import io.searchbox.core.Index;
+import io.searchbox.core.Search;
+import io.searchbox.indices.Analyze;
 import org.elasticsearch.ElasticsearchTimeoutException;
 import org.elasticsearch.action.WriteConsistencyLevel;
 import org.elasticsearch.action.admin.indices.analyze.AnalyzeResponse;
```

Comment:
Lots of unused imports and some remains from the Elasticsearch client (everything `org.elasticsearch.*` basically).